src/components/UniversalNavbarExpanded/DropdownNav/DropdownCta.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+// Reused assets from UniversalNavbar
+import { DropdownLinkIcon } from '../../UniversalNavbar/assets/icons.js'
+
+// EDS core components
+import { Spacer, Body, COLORS, TitleSmall } from '../../index'
+
+// Helpers
+import { removeLastWord, getLastWord } from '../../../helpers/splitLastWord'
+
+// Styles
+import styles from './DropdownCta.module.scss'
+
+/**
+ * CTA copy for use in DropdownNav
+ *
+ *
+ * @param {string} title - A short title
+ * @param {string} subcopy - A sentence of supporting copy
+ *
+ * @return {JSX.Element}
+ */
+const DropdownCta = ({ title, subcopy }) => (
+  <>
+    <TitleSmall.Serif.Book500>
+      <span>{removeLastWord(title)}</span>
+      <div className={styles.inlineBlock}>
+        <span>{getLastWord(title)}</span>
+        <DropdownLinkIcon />
+      </div>
+    </TitleSmall.Serif.Book500>
+    <Spacer.H8 />
+    <Body.Regular400 color={COLORS.GRAY_SECONDARY}>{subcopy}</Body.Regular400>
+  </>
+)
+
+DropdownCta.propTypes = {
+  title: PropTypes.string.isRequired,
```

Comment:
So above here we'd have:

```js
  /** `title` - (string) a short title displayed in the navbar */
```

or whatever makes sense. It ends up displayed in a table kinda like this:

![image](https://user-images.githubusercontent.com/142403/75909972-ec42cb00-5e01-11ea-9b88-51ce593a81c0.png)

Reply:
I can probably copy/paste most of the JSDoc stuff down into PropTypes, but it feels a bit redundant being in both places. 

I'm not strongly opposed, more-so curious what the benefit of having the same comments in the PropTypes are if they are already at the top of the file in JSDoc comment.

Reply:
So I think I'm just going to go with: 
```
/** Prop description */
propName: PropTypes.type
```

Based on the comparison below. 

----

```
/** `hideMobileCta` - (boolean) Hide cta on mobile */
```
![Screen Shot 2020-03-04 at 5 24 32 PM](https://user-images.githubusercontent.com/4285270/75938847-95f37d80-5e3d-11ea-997d-9c8c0bc07170.png)
```

/** Hide cta on mobile */
```

![Screen Shot 2020-03-04 at 5 26 09 PM](https://user-images.githubusercontent.com/4285270/75938845-94c25080-5e3d-11ea-91d1-f703a0488cb4.png)

----

Having the property name and type twice seems unnecessary unless I'm totally missing something. The second screenshot seems easier to read to me. @roblevintennis thoughts?

Reply:
I can also nest a little further and give a description to each of the `shape` props in `links`, but I think that's secondary to the above. 

```
  /** URLs and text */
  links: PropTypes.shape({
    /** Root application link { href: string} */
    INDEX: PropTypes.shape({
        /** `href` - (string) a url */
      href: PropTypes.string,
    }),
```

![Screen Shot 2020-03-04 at 5 37 33 PM](https://user-images.githubusercontent.com/4285270/75939389-e91a0000-5e3e-11ea-8630-c4f7a5fadfd9.png)

Reply:
Planning to resolve this in a separate PR.